mappings/net/minecraft/util/math/MathHelper.mapping

```diff
@@ -262,9 +267,16 @@ CLASS net/minecraft/class_3532 net/minecraft/util/math/MathHelper
 		ARG 1 min
 		ARG 2 max
 	METHOD method_32751 nextBetween (Ljava/util/Random;II)I
```

Comment:
I wonder if this (or maybe `nextInt`) deserves a better name at some point? This is like an "unsafe" version of `nextInt`, which is like an inclusive version of `Random.nextInt` with that extra guard for the range.